pkg/security/ebpf/c/include/hooks/network/dns.h

```diff
@@ -123,4 +123,47 @@ int classifier_dns_request_parser(struct __sk_buff *skb) {
     return ACT_OK;
 }
 
+SEC("classifier/dns_response")
+int classifier_dns_response(struct __sk_buff *skb) {
+    struct packet_t *pkt = get_packet();
+
+    if (pkt == NULL) {
+        // should never happen
+        return ACT_OK;
+    }
+
+    int len = pkt->payload_len;
+
+    if (len > DNS_MAX_LENGTH) {
+        len = DNS_MAX_LENGTH;
+    }
+
+    struct dns_response_event_t evt = {0};
```

Comment:
We talked about it last week - and definitely for a v2 - but it would be great to leverage a PER_CPU_ARRAY here so we're not constrained by the eBPF stack 😃

Reply:
Perfect ! 🎉